client/src/components/Navbar/Logo.tsx

```diff
@@ -0,0 +1,28 @@
+import { Grid, Typography } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import { useHistory } from 'react-router';
+
+const useStyles = makeStyles((theme) => ({
+  text: {
+    letterSpacing: 8,
+    color: 'white',
+    fontWeight: 'bolder',
+    fontSize: 20,
+    [theme.breakpoints.down('md')]: {
+      fontSize: 18,
+    },
```

Comment:
This is kind of an odd breakpoint change to see when resizing. I don't think it's really necessary in this case

Reply:
Thanks for the feedback. Fixed in the upcoming commit.